### PR TITLE
Fixed an issue where comments were shown as empty when being editted

### LINKED ISSUE #5318: edit comment text-box to contain text to be edited
This is a little refactor to the comment section. 

On clicking the edit icon as shown in the GIF below, it would be right for the edit text-box to contain text that is to be edited rather than have an empty text box as though creating a new comment.

The placeholder text is also a bit misleading, seems that placeholder text belongs to 'reply to this comment' text-box.

edit-bug

The target controller in this case would be the Comment Controller, specifically the update action:  https://github.com/publiclab/plots2/blob/master/app/controllers/comment_controller.rb#L108

And the target view would be: https://github.com/publiclab/plots2/blob/master/app/views/comments/_edit.html.erb

### PR DESCRIPTION
**Pull request by kylebess9**
Fixes #5318 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
**I don't know how to change branch names**
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
**I didn't change anything with the UI**

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/comments/create.html.erb [hunk 1]
```diff
@@ -4,6 +4,5 @@ $('#answer-<%= @answer_id %>-comment-<%= @comment.id %>').show()
 $('#answer-<%= @answer_id %>-comment-count')[0].innerHTML = parseInt($('#answer-<%= @answer_id %>-comment-count')[0].innerHTML) + 1
 $('#answer-<%= @answer_id %>-comment-form textarea').val('');
 $('#answer-<%= @answer_id %>-comment-section .help-block').remove();
-$('textarea').val("");
```
Comment (jywarren) [inline app/views/comments/create.html.erb]: Hi! This is looking good, but will it still clear the comment textarea where you typed? Perhaps we need to leave this here, but to link it more closely to the correct textarea, like with `#answer-<%= @answer_id %>-comment-section textarea`?

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -4,5 +4,4 @@
   $('#comment-<%= @comment.reply_to %>-reply').before('<%= escape_javascript(render :partial => "notes/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
 <% end %>
 $('#answer-<%= @answer_id %>-comment-section .help-block').remove();
-$('textarea').val("");
```
Comment (jywarren) [inline app/views/comments/create.js.erb]: and likewise here with something like `#comment-<%= @comment.reply_to %>-reply`?

Comment (jywarren) [reply app/views/comments/create.js.erb]: Do you think you could try it out on your local copy of the app? Have you been able to install it? Thanks a ton!

Comment (kylebess9) [reply app/views/comments/create.js.erb]: Yes, I will try it this weekend. Thank you for your help

Comment (SidharthBansal) [reply app/views/comments/create.js.erb]: Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab
cc/ @jywarren @gauravano

**Pull request closed without merging**